README.rst

```diff
@@ -176,17 +194,10 @@ Management of Requirements & Licenses
 =====================================
 
 Installation requirements of your project can be defined inside ``setup.cfg``,
-e.g. ``install_requires = numpy; scipy``. To avoid package dependency problems,
+e.g. ``install_requires = numpy; scipy``. To avoid package dependency problems
 it is common to not pin installation requirements to any specific version,
-although minimum versions, e.g. ``sphinx>=1.3``, or maximum versions, e.g.
-``pandas<0.12``, are used sometimes.
-
-More specific installation requirements should go into ``requirements.txt``.
-This file can also be managed with the help of ``pip compile`` from `pip-tools`_
-that basically pins packages to the current version, e.g. ``numpy==1.13.1``.
-The packages defined in ``requirements.txt`` can be easily installed with::
```

Comment:
let's remove mentions to requirements.txt from README, it might just be confusing...